docs/source/admin/traffic_vault.rst

```diff
@@ -79,6 +79,41 @@ Administration of the PostgreSQL database for Traffic Vault
 
 Similar to administering the Traffic Ops database, the :ref:`admin <database-management>` tool should be used for administering the PostgreSQL Traffic Vault backend.
 
+.. program:: reencrypt
+
+app/db/reencrypt/reencrypt
+--------------------------
+The :program:`app/db/reencrypt/reencrypt` binary is used to re-encrypt all data in the Postgres Traffic Vault with a new base64-encoded AES key.
```

Comment:
the argument to `:program:` should be the same as the argument to the `.. program::` directive; in this case: `reencryt`